PyInstaller/building/build_main.py

```diff
@@ -29,7 +29,8 @@
 from .. import compat
 from .. import log as logging
 from ..utils.misc import absnormpath, compile_py_files
-from ..compat import is_py2, is_win, PYDYLIB_NAMES, VALID_MODULE_TYPES
+from ..compat import is_py2, is_win, PYDYLIB_NAMES, VALID_MODULE_TYPES, \
+    open_file, text_read_mode, text_type, unicode_writer
```

Comment:
Just an idea: If we `import open_file as open`, we will save quite some not-meaningful changes when dropping py2-compatilitiy. What do you think?

Reply:
I'm ambivalent. As you say, this makes moving to Python 3 only easier. But, I don't like the idea of re-defining a built-in function.